app/vulncode_app/models.py

```diff
@@ -26,15 +26,21 @@
 
 
 class Vulnerability(models.Model):
-    summary = models.TextField(max_length=50, help_text="Summary of the vulnerability")
-    cvss = models.FloatField(max_length=50, help_text="CVSS Score")
+    summary = models.TextField(max_length=50, help_text="Summary of the vulnerability", null=True)
+    cvss = models.FloatField(max_length=50, help_text="CVSS Score", null=True)
+
+    def __str__(self):
+        return self.summary
 
 
 class VulnerabilityReference(models.Model):
-    vulnerability = models.ForeignKey('Vulnerability')
-    source = models.CharField(max_length=50, help_text="Source's name eg:NVD")
-    reference_id = models.CharField(max_length=50, help_text="Reference ID, eg:CVE-ID")
-    url = models.URLField(max_length=1024, help_text="URL of Vulnerability data")
+    vulnerability = models.ForeignKey('Vulnerability', null=True)
```

Comment:
What's the use case for creating VulnerabilityReference  without a Vulnerability reference?